src/qt/signverifymessagedialog.cpp

```diff
@@ -41,6 +41,9 @@ SignVerifyMessageDialog::SignVerifyMessageDialog(const PlatformStyle *_platformS
     GUIUtil::setupAddressWidget(ui->addressIn_SM, this);
     GUIUtil::setupAddressWidget(ui->addressIn_VM, this);
 
+    ui->messageIn_VM->setPlaceholderText(tr("Enter the signed message to verify"));
```

Comment:
Why aren't these in the .ui file?

Reply:
I tried to keep the changes consistent with the rest of this file, by adding the placeholder text in code

Reply:
FWIW the historical reason for this is that there was some Qt 4.x version that didn't support placeholder text in the `.ui` file. Support for Qt4 was dropped ages ago so this is not a concern anymore. Feel free to simply add it in the ui file.

Reply:
Ahh that makes sense, thank you. I'll redo these changes tonight in the UI file and commit.